cookbook-efforts/multilingual-prompt-engine/responses/dutch.md

```diff
@@ -0,0 +1,101 @@
+# Model: google/gemma-2-27b-it
```

Comment:
As mentioned in your comments, after having access to the endpoints, it would be nice to push the data to the HUB instead of saving it as .md files